src/TextMateSharp.Grammars/RegistryOptions.cs

```diff
@@ -171,6 +183,13 @@ void InitializeAvailableGrammars()
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This method will be inlined by the compiler so it won't affect the performance at all.